examples/file_based_server/server_config.json5

```diff
@@ -0,0 +1,176 @@
+// JSON 5 server configuration
+{
+  // Server Description
+  buildInfo: {
+    productUri: "build.product.test.org",
+    manufacturerName: "Manufacture GmbH",
+    productName: "Product Name",
+    softwareVersion: "0.1.0",
+    buildNumber: "0.9.4",
+    buildDate: "2022-11-02T15:31:03Z",
+  },
+  applicationDescription: {
+    applicationUri: "urn:open62541.server.application",
+    productUri: "urn:product.test.org",
+    applicationName: {
+      locale: "de-DE",
+      text: "Test text"
+    },
+    applicationType: 0,
+    gatewayServerUri: "GATEWAY",
+    discoveryProfileUri: "DISC",
+    discoveryUrls: [
+      "__open",
+      "_62541",
+      "opc ua"
+    ],
+  },
+  shutdownDelay: 5000.50,
+
+  verifyRequestTimestamp: 0,
+  allowEmptyVariables: 0,
+
+  // Networking
+  // Set up the local ServerUrls. They are used during startup to initialize the server sockets.
+  serverUrls: [
+    "opc.tcp://127.0.0.1:4840",
+  ],
+  tcpBufSize: 64000,
+  tcpMaxMsgSize: 0,
+  tcpMaxChunks: 0,
+
+  // Limits for SecureChannels
+  maxSecureChannels: 10,
+  maxSecurityTokenLifetime: 300000,
+
+  // Limits for Sessions
+  maxSessions: 50,
+  maxSessionTimeout: 3600000,
+
+  // Operation limits
+  maxNodesPerRead: 10000,
+  maxNodesPerWrite: 10000,
+  maxNodesPerMethodCall: 10000,
+  maxNodesPerBrowse: 10000,
+  maxNodesPerRegisterNodes: 10000,
+  maxNodesPerTranslateBrowsePathsToNodeIds: 10000,
+  maxNodesPerNodeManagement: 10000,
+  maxMonitoredItemsPerCall: 10000,
+
+  // Limits for Requests
+  maxReferencesPerNode: 0,
+
+  // Limits for Async Operations
+  asyncOperationTimeout: 120000,
+  maxAsyncOperationQueueSize: 1000000,
+
+  // Discovery Multicast
+  mdnsEnabled: false,
+  mdnsConfig: {
+    mdnsServerName: "mdnsServerName",
+    serverCapabilities: [
+      "server",
+      "capa",
+      "bilities"
+    ]
+  },
+  mdnsInterfaceIP: "127.0.0.1",
+  mdnsIpAddressList: [
+    1,
+    2,
+    3
+  ],
+
+  // Limits for Subscriptions
+  maxSubscriptions: 100,
```

Comment:
Add an object `subscriptions` to regroup the following items.
Also the monitoreditem settings.
At some point we might have `enableSubscriptions` as a config option instead of a compile flag.